src/anemoi/inference/runners/parallel.py

```diff
@@ -341,14 +348,16 @@ def _init_network_from_slurm(self) -> Tuple[str, str]:
 
         return master_addr, master_port
 
-    def _init_parallel(self) -> Optional[dist.ProcessGroup]:
+    def _init_parallel(self) -> Optional["torch.distributed.dist.ProcessGroup"]:
```

Comment:
Should be `torch.distributed.ProcessGroup`

Reply:
Fixed